docs/usage/administration/admin_faq.md

```diff
@@ -70,13 +70,54 @@ output-directory
 │       ├───state
 │       ├───invite_state
 │       └───knock_state
-└───user_data
-    ├───account_data
-    │   ├───global
-    │   └───<room_id>
-    ├───connections
-    ├───devices
-    └───profile
+├───user_data
+│   ├───account_data
+│   │   ├───global
+│   │   └───<room_id>
+│   ├───connections
+│   ├───devices
+│   └───profile
+└───media_ids
+    └───<media_id>
+```
+
+The `media_ids` folder contains only the metadata of the media, not the media itself.
```

Comment:
I think this needs to say that it is the data the user uploaded. (Which I'm not even sure is correct -- is that what this is?)

Reply:
Yes, this is from media uploaded by this user.